internal/serverinstall/k8s.go

```diff
@@ -956,6 +956,20 @@ func (i *K8sInstaller) InstallFlags(set *flag.Set) {
 			"The required version number format is: 'vX.Y.Z'.",
 	})
 
+	set.IntVar(&flag.IntVar{
+		Name:    "k8s-helm-timeout",
+		Target:  &i.Config.Timeout,
+		Default: 300,
+		Usage:   "Time to wait in seconds for the Helm operation.",
+	})
+
+	set.BoolVar(&flag.BoolVar{
+		Name:    "k8s-helm-atomic",
+		Target:  &i.Config.Atomic,
+		Default: false,
+		Usage:   "Make the Helm operation atomic.",
```

Comment:
Mind expanding this description?

Reply:
you can snag the output from helm, like:
```
if set, upgrade process rolls back changes made in case of failed upgrade. The --wait flag will be set automatically if --atomic is used
```